webapp/utils/async_client.jsx

```diff
@@ -131,7 +131,6 @@ export function updateLastViewedAt(id) {
         () => {
             callTracker.updateLastViewed = 0;
             ErrorStore.clearLastError();
-            ErrorStore.emitChange();
```

Comment:
updateLastView is called every 30s so if the server is down the error will appear.  When the server is backup the error should disappear on next successful reconnect.